Prometheus/confd/templates/prometheus.yml.tpl

```diff
@@ -37,7 +37,6 @@ scrape_configs:
     static_configs:
     - targets:
         - https://bichard-web.{{getv "/cjse/fqdn/suffix" "cjse.org"}}/bichard-ui/Health
```

Comment:
the link in the first modified file has the prefix: "bichard-backend". should this be updated as well?

Reply:
Nope, this is a different check